harvard-lts/jstor_aspace_transformer
# Ss 245

**Create delete/drop records for all cases**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-245)

# What does this Pull Request do?
Pushes delete and drop records through the xslt process; handles all cases:
- oai deletes (delete only in oai header)
-  delete because all display records are send to harvard false
- delete because all display records are status deleted
- drop record because no work record (only display rec)

We now keep these records in the context of the set it was processed in (needed for reporting)

This positions the publisher to process and report on deletes/drops (SS-246 and SS-251)

# How should this be tested?
Unit tests:
Check out this branch locally
Run docker container locally
Exec into docker: > docker exec -it [container-id] bsh
From inside docker run > pytest

Deploy to dev and ITests should pass (done)

Run tests cases in the observatory set and confirm they pass through (done)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests?

# Interested parties
Tag (@ mention) interested parties
